libkbfs/bare_root_metadata_v3.go

```diff
@@ -397,7 +395,18 @@ func (md *BareRootMetadataV3) IsWriter(
 		if err != nil {
 			return false, err
 		}
-		return wkb.IsWriter(user, deviceKey), nil
+		return wkb.IsWriter(user, cryptKey), nil
+	default:
+		panic(fmt.Sprintf("Unknown TLF type: %s", md.TlfID().Type()))
```

Comment:
I didn't introduce this, it already existed before.  I could fix it, but it seems like a panic on the client side is a good thing, to make it obvious when we miss adding cases to `select` like this.

However I do see it's used in one place on the server side, though only for public TLFs.  I guess I'll make it an error and hope that we'll be able to catch all these cases through simple errors next time.